scripts/utils.py

```diff
@@ -67,34 +73,50 @@ def load_mesh_mapping_file(mesh_mapping_file_path) -> dict:
   with open(mesh_mapping_file_path, 'r') as f:
     return json.load(f)
 
-def map_mesh_to_values(mesh_code, loaded_mesh_mapping) -> list:
+def map_mesh_to_values(mesh_code, loaded_mesh_mapping):
   for term in loaded_mesh_mapping:
     if term['code'] == mesh_code: return term['name']
 
-def map_values_to_mesh(value, url = "https://goldorak.hesge.ch:8082/transmesh/translate/"):
+def map_values_to_mesh(value, url = "https://goldorak.hesge.ch:8082/transmesh/translate/", banned_columns=None):
   assert type(value)==str, "Value passed to 'map_values_to_mesh' must be of type <str>"
   data = {
-    'term': value,
-    'terminology': 'text',
-    'result_format': 'json'
-    }
+      'term': value,
+      'terminology': 'text',
+      'result_format': 'json'
+      }
   response = requests.post(url, data=data)
   maximum_ed = None
   mesh_code = None
-  for mesh_candidate in response.json(): 
-    current_ed = lcs(value, mesh_candidate['mesh_label'])
-    if maximum_ed is None or current_ed > maximum_ed:
-      maximum_ed = current_ed 
-      mesh_code = mesh_candidate['mesh_code']
-  if mesh_code is not None:
-    return mesh_code
+  if banned_columns is None:
```

Comment:
This if/else can be merge to one piece of code. Just have one loop and inside check for `if mesh_candidate['mesh_code'] not in banned_columns:`. There is a lot of duplicated code (the other parts are identical expect the check in banned_columns).